biojava-alignment/src/test/java/org/biojava/nbio/alignment/CookbookMSAProfiler.java

```diff
@@ -105,27 +109,27 @@ public static void main(String[] args) throws Exception {
 
 		profiler.setCheckpoint();
 
-		System.out.print("Stage 2: hierarchical clustering into a guide tree... ");
+		LOGGER.info("Stage 2: hierarchical clustering into a guide tree... ");
 		GuideTree<ProteinSequence, AminoAcidCompound> tree = new GuideTree<ProteinSequence, AminoAcidCompound>(list,
 				scorers);
 		scorers = null;
-		System.out.printf("%d ms using %d kB%n%n%s%n%n", profiler.getTimeSinceCheckpoint()/1000000,
-				profiler.getMaxMemoryUsed()/1024, tree);
+		LOGGER.info(String.format("%d ms using %d kB%n%n%s%n%n", profiler.getTimeSinceCheckpoint()/1000000,
+				profiler.getMaxMemoryUsed()/1024, tree));
 
 		profiler.setCheckpoint();
 
-		System.out.print("Stage 3: progressive alignment... ");
+		LOGGER.info("Stage 3: progressive alignment... ");
 		Profile<ProteinSequence, AminoAcidCompound> msa = Alignments.getProgressiveAlignment(tree,
 				ProfileProfileAlignerType.GLOBAL, gaps, blosum62);
-		System.out.printf("%d profile-profile alignments in %d ms using %d kB%n%n", list.size() - 1,
-				profiler.getTimeSinceCheckpoint()/1000000, profiler.getMaxMemoryUsed()/1024);
+		LOGGER.info(String.format("%d profile-profile alignments in %d ms using %d kB%n%n", list.size() - 1,
+				profiler.getTimeSinceCheckpoint()/1000000, profiler.getMaxMemoryUsed()/1024));
 		fout.print(msa);
 		fout.close();
 
 		ConcurrencyTools.shutdown();
 
-		System.out.printf("Total time: %d ms%nMemory use: %d kB%n", profiler.getTimeSinceStart()/1000000,
-				profiler.getMaxMemoryUsed()/1024);
+		LOGGER.info(String.format("Total time: %d ms%nMemory use: %d kB%n", profiler.getTimeSinceStart()/1000000,
+				profiler.getMaxMemoryUsed()/1024));
 
 	}
 
```

Comment:
yea, I agree, and I've followed what you say, but this one, and I already reverted it.